shell/platform/tizen/tizen_input_method_context.cc

```diff
@@ -82,6 +82,22 @@ Ecore_IMF_Keyboard_Locks EcoreInputModifiersToEcoreImfLocks(
   return static_cast<Ecore_IMF_Keyboard_Locks>(locks);
 }
 
+template <typename T>
+T NUIEventKeyToEcoreImfEvent(const char* key,
+                             const char* string,
+                             uint32_t modifiers,
+                             uint32_t scan_code) {
+  T imf_event;
+
+  imf_event.key = key;
+  imf_event.string = string;
+  imf_event.modifiers = EcoreInputModifiersToEcoreImfModifiers(modifiers);
+  imf_event.locks = EcoreInputModifiersToEcoreImfLocks(modifiers);
+  imf_event.keycode = scan_code;
```

Comment:
EcoreEvent passes more information than this.
But I don't know why they are needed.
When I tested, only `key` and `string` were sufficient.
(ewk webview only uses `key` and `string`. (evas event))
https://github.com/flutter-tizen/plugins/pull/400/files/50ac06ef0278cdf8ab3d814ab3aee7d0fabe6135#diff-c41c7de9898e3e8e225393fb4381982fb6248999451c306948584e1c330d10afR289)

If we need more, I can get it from NUI, but there is no API to get compose and time stamp information in NUI's key event.